invenio_oauthclient/handlers.py

```diff
@@ -234,20 +235,18 @@ def authorized_signup_handler(resp, remote, *args, **kwargs):
 
         if user is None:
             # Auto sign-up if user not found
-            user = oauth_register(account_info)
-            if user is None:
```

Comment:
@lnielsen it looks correct since @hachreak added https://github.com/inveniosoftware/invenio-oauthclient/pull/54/files#diff-63911819994c3bdd7c57acc33a482540R380